docs/source/conf.py

```diff
@@ -15,16 +15,19 @@
 sys.path.insert(0, os.path.abspath('../..'))
 
 import sphinx_rtd_theme
+import lightly
```

Comment:
are you sure this doesn't import an installed version of lightly? which might mess up the version... so the procedure would be 
1. install lightly from source
2. release the docs

Reply:
The `sys.path.insert(0, os.path.abspath('../..'))` makes sure that lightly is imported from the local directory and not the installed package.

Reply:
ah cool, didn't see that